### ISSUE TITLE
hacktoberissues must have "hacktoberfest" topic for the repo

### ISSUE THREAD

**Issue by hedythedev** [python-discord/seasonalbot#489]
## Description
`hacktoberissues` command returns issues, but the repo should also have "hacktoberfest" topic for it to count (and i don't really think a hacktoberfest label is needed but i may be wrong, please let me know)

with current logic, it finds issues with hacktoberfest label, without checking if the repo is even eligible 

## Possible Solutions
loop through results, only include those that has hacktoberfest label

## Would you like to implement a fix?

***Note: For high-priority or critical bugs, fixes may be implemented by staff.***

- [x] I'd like to implement the bug fix (the actual implementation and logic may need to be confirmed first)
- [ ] Anyone can implement the bug fix

**Comment by ChrisGallardo** [comment]
Yes, it's not just the `hacktoberfest` topic, but also issues labeled`hacktoberfestissue`.

**Comment by hedythedev** [comment]
@ChrisGallardo @Den4200 
i'm talking about fetching *issues* (.hacktoberissues) command, not PRs, so I don't think #475 fixes this, and I don't think `hacktoberfest-accepted` label on PRs is related. Am I missing something?

**Comment by Den4200** [comment]
@hedythedev Right, sorry, I misread the issue.

**Comment by ChrisGallardo** [comment]
Oh I see.

**Comment by Xithrius** [comment]
@hedyhli Greetings. I seem to be around a year late to this, but would you still like to implement this fix?

**Comment by hedyhli** [comment]
@Xithrius I'm not sure I'll have the time, and I haven't worked with the codebase for quite a while so I might need some getting use to.

Also (for hacktoberfest stats/prs), this year's hacktoberfest (if it's happening) probably doesn't need the "first 3 days counts even if it doesn't haven hacktoberfest label" anymore. Again, I haven't been on discord for a long time nor have I seen the commits and stuff so I'm not sure if that needs to be changed before October.

In any case, I think we should let this issue be open to others, though if I'm lucky I might have time to have a look at it this weekend.

**Comment by Xithrius** [comment]
Alright, I'll leave it up for grabs for now.

**Comment by hedyhli** [comment]
I can work on this but it's blocked by #919 for now
